app/src/main/java/de/tum/in/tumcampusapp/component/ui/news/KinoDetailsFragment.java

```diff
@@ -171,17 +171,14 @@ public void onError(Exception e) {
 
         rootView.addView(headerView);
     }
-    //open show_ticket activity, and transfer current movie data to it.
-    //The String data is just a example, it should be get from correspongding movie which now show on the screen.
-    //time and place of movie  is fixed. Only the movie title and date should be transfered to show_ticket activity
-    private void showTicket(){
-        String data = "KingsMan 08.05 " + "\n"+ "Filmbegin: 20:00 o'clock "+"\n"+ " 1. Stock, Hörsaal 1200 (Carl-von-Linde-Hörsaal) Arcisstraße 21";
-        Intent intent = new Intent(getActivity().getApplicationContext(), show_ticket.class);
-        intent.putExtra("movie_data", data);
+
+    //open ShowTicketActivity activity
+    private void showTicket() {
+        Intent intent = new Intent(getActivity().getApplicationContext(), ShowTicketActivity.class);
```

Comment:
I am not completely sure, but can't we just use the class attribute context here instead of getActivity().getApplicationContext()?

Reply:
Thank you. You are right. I fixed all the problem and make another commit